llvm/include/llvm/Analysis/LoopAccessAnalysis.h

```diff
@@ -304,6 +310,10 @@ class MemoryDepChecker {
   /// restrictive.
   uint64_t MaxSafeVectorWidthInBits = -1U;
 
+  /// Maximum power-of-2 number of elements, which do not prevent store-load
+  /// forwarding and safe to operate simultaneously.
```

Comment:
```suggestion
  /// Maximum power-of-2 number of elements, which do not prevent store-load
  /// forwarding.
```

Now just for store-load forwarding prevention?